### ISSUE TITLE
Call Forward bug

### ISSUE THREAD

**Issue by daniel-lucio** [fusionpbx/fusionpbx#2522]
2017-04-19 17:33:21.001297 [ERR] switch_regex.c:83 Regular Expression Error expression[/^(933] missing ending '/' delimeter
2017-04-19 17:33:21.001297 [ERR] switch_regex.c:83 Regular Expression Error expression[/^(933] missing ending '/' delimeter
2017-04-19 17:33:21.001297 [ERR] switch_regex.c:83 Regular Expression Error expression[/^(933] missing ending '/' delimeter
2017-04-19 17:33:21.001297 [ERR] switch_regex.c:83 Regular Expression Error expression[/^(933] missing ending '/' delimeter
2017-04-19 17:33:21.001297 [ERR] switch_regex.c:83 Regular Expression Error expression[/^(933] missing ending '/' delimeter
2017-04-19 17:33:21.001297 [ERR] switch_regex.c:83 Regular Expression Error expression[/^(933] missing ending '/' delimeter
2017-04-19 17:33:21.001297 [DEBUG] switch_cpp.cpp:1365 [route_to_bridge] PASS condition destination_number(^(933|911)$) to `2136232424`

It happens when evaluating regular expressions like ^(911|933)$
I have written a full explanation here http://inside-out.xyz/technology/call-forward-bug-in-fusionpbx-4-2.html with two possible workarounds.

In short, it seems that resources/functions/route_to_bridge.lua is not processing correctly the regexp.
